inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$ref": "#/definitions/Epinow2",
+    "definitions": {
+        "Epinow2": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "job_id": {
```

Comment:
My only slight hesitation with UUIDs for the job IDs is if we run multiple jobs. It would just be a bit harder to know which job is which. That said, it would mean we never run into the annoying error "This job already exists" because we forgot to delete it.

What about, e.g. `Rt-estimation-2024-08-08T10:08:34` as job name?

Reply:
This assumes that you will pass in a UUID that is generated somewhere else right? 

Probably not for this PR, I would add more metadata. For example, if this job id is under the "EpiNow2" umbrella, I would want something that names the job based on the name of the package being used.

Reply:
I was testing out this naming scheme idea on something else, and discovered that Azure was not happy with `:`, so I replaced it with `-`. 

So this might be something more like `Rt-estimation-2024-08-08T10-08-34`